### ISSUE TITLE
Flex theme is not able to show results from tipue_search plugin

### ISSUE THREAD

**Issue by andreagrandi** [alexandrevicenzi/Flex#251]
## Description of the issue

I've an "up and running" blog based on Pelican and I currently use tipue_search plugin. On my current theme, tipue works as expected, with Flex it doesn't.

## What I've done

- I'm using Pelican 4.5
- I'm using latest Flex 2.3
- I'm using the tipue_search plugin linked in submodules of Flex (the one generating the *.js not the old one generating the *.json)

## What happens

- `tipuesearch_content.js` is correctly generated in my `/output` folder
- all the required js, css files are correctly loaded in the Search page (I don't see any 404)
- `tipuesearch_content.js` is correctly loaded and I can inspect its response
- in the title of the page I see a number like `(15)` showing the number of results (I've checked it's the number of results because it's the same number I see in the working theme and it changes if I change the search term), so it means that somehow the search is working
- the HTML DIV `tipue_search_content` doesn't seem to be rendered properly:

<img width="753" alt="Screenshot 2020-10-04 at 21 52 02" src="https://user-images.githubusercontent.com/636391/95025971-2755f700-068e-11eb-9860-a319fa65aaee.png">

In comparison, this is what I see with my current theme (the one where tipue works)

![Screenshot 2020-10-04 at 21 50 21](https://user-images.githubusercontent.com/636391/95025985-3c328a80-068e-11eb-99c8-b6af341d78dc.png)

## Questions

- Am I using it wrong or there is some bug? 
- How can I help you to triage this issue?
- Is there any other information I can provide that could be helpful to understand the issue?

Thanks 🙌

**Comment by andreagrandi** [comment]
I've an update: I've tried to use my old tipue js files (which, for reference, you can find here https://github.com/andreagrandi/andreagrandi.it/tree/master/themes/pelican-blueidea/static/tipuesearch but it should be 6.1 version). 

I've updated the imports in `base.html` (the old version uses a non minified version):

```
<!-- <script src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/tipuesearch/tipuesearch_set.min.js"></script> -->
<script src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/tipuesearch/tipuesearch_set.js"></script>
```

and the search results are rendered just fine.

Now, I've no idea why the old version works and the new one doesn't but I hope this can be an additional help to understand the issue.

**Comment by sercxanto** [comment]
@andreagrandi FYI I got tipue search up and running with pelican 4.7.1, Flex 2.4.0 and the latest tipue plugin ,see  #291 .

**Comment by alexandrevicenzi** [comment]
This should be fixed by #291.

**Comment by alexandrevicenzi** [comment]
Also, note that tipue search is deprecated and may stop working with newer releases of Pelican. In that case, it may be dropped from this theme.
